### PR TITLE
Update font-genyogothic.rb to `v2.000`

### PR DESCRIPTION
**Pull request by singularitti**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/font/font-g/font-genyogothic.rb [hunk 1]
```diff
@@ -1,18 +1,20 @@
 cask "font-genyogothic" do
-  version "1.501"
-  sha256 "d90705ecb3ab25a9b98972fcb244323b6a0c76c4320bb7e0ab1b82815f727086"
+  version "2.000"
+  sha256 "e570102fd1c9abb67897b215e6b7f2a0666610f9bb1e2af5fbb3a2db8f67fa93"
 
-  url "https://github.com/ButTaiwan/genyog-font/releases/download/v#{version}/GenYoGothic.zip"
+  url "https://github.com/ButTaiwan/genyog-font/releases/download/v#{version}/GenYoGothic#{version.major}.zip"
```
Comment (khipp) [inline Casks/font/font-g/font-genyogothic.rb:5]:
```suggestion
  url "https://github.com/ButTaiwan/genyog-font/releases/download/v#{version}/GenYoGothic#{version.major}-ttc.zip"
```

Comment (singularitti) [reply Casks/font/font-g/font-genyogothic.rb:5]: that's an overlook, thanks!

**Comment by singularitti** [comment]
Hi, I forgot to mention it is a traditional Chinese font in the description, could you help me update it? Thank you. @khipp

File: Casks/font/font-g/font-genyogothic.rb [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
-  version "1.501"
-  sha256 "d90705ecb3ab25a9b98972fcb244323b6a0c76c4320bb7e0ab1b82815f727086"
+  version "2.000"
+  sha256 "e570102fd1c9abb67897b215e6b7f2a0666610f9bb1e2af5fbb3a2db8f67fa93"
 
-  url "https://github.com/ButTaiwan/genyog-font/releases/download/v#{version}/GenYoGothic.zip"
+  url "https://github.com/ButTaiwan/genyog-font/releases/download/v#{version}/GenYoGothic#{version.major}-ttc.zip"
   name "GenYoGothic"
+  name "源樣黑體"
+  desc "Open-source Chinese font derived from Source Han Sans"
```
Comment (khipp) [inline Casks/font/font-g/font-genyogothic.rb:8]:
```suggestion
```

Thank you for making me aware of this change. Unfortunately, we have recently decided to omit `desc` stanzas from font casks (see https://github.com/Homebrew/homebrew-cask/pull/176580).

**Review by khipp — APPROVED** [review]
Thank you, @singularitti!

**Comment by singularitti** [comment]
Thank you!

**Pull request merged**